qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -94,6 +94,12 @@ def statfs(self, path):
         self.assertIsInstance(data, dict)
         return data
 
+    def rename(self, src_path, dst_path):
+        params = {'src_path': src_path, 'dst_path': dst_path}
+        self._put(f"/api/cephfs/{self.get_fs_id()}/rename-path",
```

Comment:
`rename-path` looks invalid, looks like typo?

Reply:
I have confirmed multiple times that there are no typo. I don't know yet why the API test failed here.

Reply:
shouldn't this be `rename_path`?

Reply:
no, is `rename-path`

Reply:
okay i hope this is correct since the func is named `rename_path`.

Reply:
don't worry, this is openapi check automatically generate